vuu-ui/packages/vuu-shell/src/layout-management/useLayoutManager.tsx

```diff
@@ -99,26 +111,42 @@ export const LayoutManagementProvider = (
     [setApplicationLayout]
   );
 
-  const saveLayout = useCallback((metadata: LayoutMetadataDto) => {
-    const layoutToSave = resolveJSONPath(
-      applicationLayoutRef.current,
-      "#main-tabs.ACTIVE_CHILD"
-    );
-
-    if (layoutToSave) {
-      getPersistenceManager()
-        .createLayout(metadata, layoutToSave)
-        .then((metadata) => {
-          //TODO: Show success toast
-          setLayoutMetadata((prev) => [...prev, metadata]);
-        })
-        .catch((error: Error) => {
-          //TODO: Show error toaster
-          console.error("Error occurred while saving layout", error);
+  const saveLayout = useCallback(
+    (metadata: LayoutMetadataDto) => {
+      const layoutToSave = resolveJSONPath(
+        applicationLayoutRef.current,
+        "#main-tabs.ACTIVE_CHILD"
+      );
+
+      if (layoutToSave) {
```

Comment:
It seems like we shouldn't be getting the issue of an empty JSON being saved with this guard clause here - unless it's merely an undefined check, not an empty check?

Not necessarily relevant to this PR, but if the quick fix is to add an additional 'not empty' check it might be worth doing.

Reply:
I think this is an undefined check, because `resolveJSONPath` returns `LayoutJSON | undefined` I think it was @vferraro-scottlogic who first set up this flow. Not sure I fully understand your comment though - is empty JSON an issue we are facing?

Reply:
correct. the truthy check there is needed because resolveJSONPath() could return undefined and in that case we can't call createdLayout() (removing the check results in the compiler complaining on line 123)

Reply:
note that an empty object is not falsy so that check would pass for {}

Reply:
Ticket created #117

Reply:
As discussed, this _might_ be worth doing, just under a different ticket.